packages/astro/src/assets/services/vendor/squoosh/codecs.ts

```diff
@@ -291,6 +291,7 @@ export const codecs = {
   avif: {
     name: 'AVIF',
     extension: 'avif',
+    // eslint-disable-next-line no-control-regex, regexp/control-character-escape
```

Comment:
Can we add a reason that explains the suppression?

Reply:
Will do. Mainly here I didn't want to touch vendored source code.